### PR TITLE
Translate 2014/09/10/confoo-cfp/ (zh_cn)

### PR DESCRIPTION
**Pull request by Zane5**
screen shot 2015-06-02 at 12 18 56

Related [#1074](https://github.com/ruby/www.ruby-lang.org/issues/1074)

### CODE REVIEW

File: zh_cn/news/_posts/2014-09-10-confoo-cfp.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: news_post
+title: "ConFoo 2015 is looking for Ruby speakers"
+author: "ylarrivee"
+translator:
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-09-10-confoo-cfp.md]: Put your name here please.

File: zh_cn/news/_posts/2014-09-10-confoo-cfp.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+date: 2014-09-10 06:00:00 +0000
+lang: en
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-09-10-confoo-cfp.md]: Specify lang to `zh_cn`

Comment (Zane5) [inline zh_cn/news/_posts/2014-09-10-confoo-cfp.md]: I have add the translator and the lang.
thanks for your patience.

**Comment by JuanitoFatas** [comment]
Thank you. Merged manually as 4cf1deb.

**Pull request closed without merging**